test/test_lldp.cpp

```diff
@@ -59,3 +59,36 @@ TEST_F(LldpTest, LldpInitTest)
    EXPECT_EQ(appdata.call_counters.write_calls, 0);
    EXPECT_EQ(mock_os_data.eth_send_count, 1);
 }
+
+TEST_F(LldpTest, LldpGenerateAliasName)
+{
+   char alias[100];
+   int err; 
+
+   err =  pf_lldp_generate_alias_name(NULL, "dut", alias , 96);
+   EXPECT_EQ(err, -1);
+   
+   err =  pf_lldp_generate_alias_name("port-001", NULL, alias , 96);
+   EXPECT_EQ(err, -1);
+
+   err =  pf_lldp_generate_alias_name("port-001", "dut", NULL , 96);
+   EXPECT_EQ(err, -1);
+
+
+   err =  pf_lldp_generate_alias_name("port-001", "dut", alias, 13);
+   EXPECT_EQ(err, 0);
+   EXPECT_EQ(strcmp(alias, "port-001.dut"), 0);
+   memset(alias, 0, sizeof(alias));
+
+   err =  pf_lldp_generate_alias_name("port-001", "dut", alias, 12);
+   EXPECT_EQ(err, -1);
+
```

Comment:
Add tests for Profinet 2.2 legacy devices as well as Profinet 2.3+ devices.

Reply:
I have added some comments and a test based on test data from rt-tester